### PR TITLE
Set map max bounds

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

File: src/components/LeafletMap.js [hunk 1]
```diff
@@ -100,7 +100,8 @@ const mapProperty = {
   minZoom: 2,
   maxZoom: 16,
   zoomControl: true,
-  layers: [worldImageryMapLayer, labelLayer]
+  layers: [worldImageryMapLayer, labelLayer],
+  maxBoundsViscosity: 0.5
```
Comment (brianbancroft) [inline src/components/LeafletMap.js]: Is this being used?

Comment (tienhoah) [reply src/components/LeafletMap.js]: the maxBoundsViscoscity is used to control the bounce strength from 0 to 1. I found the default a very bouncy. so I just set half

Comment (brianbancroft) [reply src/components/LeafletMap.js]: Cool :taco:

**Comment by tienhoah** [comment]
Change made in pr 15

**Pull request closed without merging**